modules/tinymce/src/themes/silver/main/ts/ui/header/StickyHeader.ts

```diff
@@ -53,6 +53,7 @@ const scrollFromBehindHeader = (e: ScrollIntoViewEvent, containerHeader: SugarEl
 };
 
 const isDockedMode = (header: AlloyComponent, mode: 'top' | 'bottom') => Arr.contains(Docking.getModes(header), mode);
+const isFullscreen = (editor: Editor) => editor.hasPlugin('fullscreen') && editor.plugins.fullscreen.isFullscreen();
```

Comment:
We already have one of these in silver, can you just make it an `export` and use it here?
https://github.com/tinymce/tinymce/blob/e8e29c60b01dcb9a72e907361235913edb3da9fb/modules/tinymce/src/themes/silver/main/ts/modes/ScrollingContext.ts#L31-L32